README.md

```diff
@@ -864,3 +864,14 @@ enableSearch = true
     Live search works even for multilingual sites.
 
     For Chinese-like languages, it may or may not work.
+
+__Search Scope__
+
+- Searching within a section will yield results from that section.
+
+    For example, if you have 2 `mainSections` 1/ vegetables 2/ fruits, searching in the fruits section will only produce results for that section.
+- Searching outside a section will search the entire site.
+
+     For example, with the above `mainSections` setup, searching from the homepage will produce results from both sections and other pages of the site.
+
+     Future implementations may include a github-like feature i.e `search all site`. This PR will leave that to a future PR
```

Comment:
I'm not sure I'd put this second sentence in the readme itself. I think it's enough to say that this is planned.

Reply:
Agree with ☝️

Reply:
@rootwork, I modified the readme in bd7752d3d7f0bb7c34ab518cc4984c0af7c44ddc. I also opened #308 to track the additional feature